lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="pagination-wrapper" [ngClass]="className" id="id">
```

Comment:
Shouldn't this be one-way binding?
```suggestion
<div class="pagination-wrapper" [ngClass]="className" [id]="id">
```